pom.xml

```diff
@@ -192,6 +192,23 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>5.5.0</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>net.bytebuddy</groupId>
+            <artifactId>byte-buddy</artifactId>
+            <version>1.14.6</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>net.bytebuddy</groupId>
+            <artifactId>byte-buddy-agent</artifactId>
```

Comment:
Are byte-buddy and the agent really necessary? I thought Mockito brings it in transitively.
If it is, it would at least be nice to explicitly set the scope to `test` so it doesn't accidentally get shaded in or anything.

Reply:
I must've forgotten setting it to "test" 😄

Yep, byte-buddy is a transitive, but there's a versioning issue (I don't remember the right Exception), so we're basically just specifying the version here.